app/konnect/vitals/custom-reports.md

```diff
@@ -0,0 +1,115 @@
+
+    All time frames are static. The report will capture a snapshot of data
+    in whichever time frame you choose, but the range always stops at the time
+    that the report was created. You can see a preview of the exact range below
+    the time frame selector. For example:
+
+    ```
+    2022-01-27T21:30:58Z to 2022-02-03T21:30:58Z
+    ```
+
+
+1. Choose a report type.
+
+   * **Service report**: Generate a report based on Services.
+   * **Route report**: Generate a report based on Routes.
+   * **Application report**: Generate a report based on API consumers.
+
+   Depending on the report type you choose, the available metrics and entities
+   will change.
+
+1. Choose a [metric](#metrics) to group the data by.
+1. Choose the entities to focus on in your report.
+1. Click **Create**.
+
+The report details page opens. Here you can:
+
+* **View details**: Hover over a bar on the graph to see details about the data.
+* **Filter the graph**: Click on an item in the legend to temporarily hide it from view,
+and click it again to show the item. The graph resets on a refresh.
+
+## Export a custom report
+
+You can export any custom report in CSV format.
+
+1. From the left-side menu in Konnect, open ![](/assets/images/icons/konnect/icn-vitals.svg){:.inline .no-image-expand}
+**Vitals**, then **Reports**.
+1. Click on a report row to open the report's page.
+1. Click the **Actions** dropdown and select **Export Report**.
+
+  Konnect generates a CSV file download with all the data in the report.
+
+## Edit a custom report
+
+You can edit a report in one of the following ways:
+
+1. From the left-side menu in Konnect, open ![](/assets/images/icons/konnect/icn-vitals.svg){:.inline .no-image-expand}
+**Vitals**, then **Reports**.
+1. Either:
+   * Click the ![](/assets/images/icons/konnect/konnect-settings.svg){:.inline .no-image-expand}
+   settings icon for a report row and select **Edit**.
+   * Click on a report row to open the report. In the **Actions** dropdown menu,
+   select **Edit Report**.
+1. Make your changes and click **Update**.
+
+  The report page opens. You may have to refresh the page to see the updated
+  report.
+
+## Delete a custom report
+
+You can delete a report in one of the following ways:
+
+1. From the left-side menu in Konnect, open ![](/assets/images/icons/konnect/icn-vitals.svg){:.inline .no-image-expand}
+**Vitals**, then **Reports**.
+1. Either:
+   * Click the ![](/assets/images/icons/konnect/konnect-settings.svg){:.inline .no-image-expand}
+   settings icon for a report row and select **Delete**.
+   * Click on a report row to open the report. In the **Actions** dropdown menu,
+   select **Delete Report**.
+1. Confirm deletion in the dialog.
+
+## Metrics
+
+Each metric depends on a time frame and a primary entity (report type).
+
+Metric | Report type | Description
+-------|------------|------------
+Total traffic | Service, Route, Application | The total number of API calls within the selected time frame.
```

Comment:
```suggestion
Total traffic | Service, Route, Application | Total number of API calls within the selected time frame.
```

Matching the description start of the content that comes after: "Number of API calls..." not "The number of API calls..."